### ISSUE TITLE
Collect(<youtube>) and Collect(<thread.reddit>) throw an error in R4.4.0: object 'collect_log' not found

### ISSUE THREAD

**Issue by sbinn** [vosonlab/vosonSML#56]
On R 4.4.0 with vosonSML 0.34.2, `Collect(<youtube>)` and `Collect(<thread.reddit>)` throw the errors below, respectively: 

Error in Collect.youtube(yt_auth, videoIDs = video_url, maxComments = 500,  : 
  object 'collect_log' not found

Error in Collect.thread.reddit(Authenticate("reddit"), threadUrls = thread_urls,  : 
  object 'collect_log' not found

Sample:
```
# YouTube Data Collection --------------------------------------------------
library(vosonSML)
my_api_key <- "XXX"
yt_auth <- Authenticate("youtube", apiKey = my_api_key)
video_url <- c("https://www.youtube.com/watch?v=jNQXAC9IVRw")
yt_data <- yt_auth |> Collect(videoIDs = video_url,
                              maxComments = 500,
                              writeToFile = TRUE,
                              verbose = TRUE) 

# Reddit Data Collection --------------------------------------------------
thread_urls <- c(
  "https://www.reddit.com/r/AusFinance/comments/ymkrlr/",
  "https://www.reddit.com/r/AusFinance/comments/ugetai/")
rd_data <- Authenticate("reddit") |>  
  Collect(threadUrls = thread_urls,
          sort = "best", 
          waitTime = c(6, 8),
          writeToFile = TRUE, 
          verbose = TRUE) 
```

**Comment by bryn-g** [comment]
Thanks @sbinn, this appears to be a new issue with `R 4.4` and `S3` classes and/or global variables as used with the metadata logging. It does not occur under `R 4.3`.

I've disabled the `writeToFile` metadata logging in `vosonSML 0.34.3` commit d19d0a5 for the time being until the issue is determined and solved. You may still encounter the problems I have with Issue #58.

**Comment by sbinn** [comment]
Thanks for the commit. For YouTube, `vosonSML 0.34.3` on `R 4.4.0` fixes the issue for the `Collect()` function. However, it throws an error when trying to create an `igraph` object from a `vosonSML` network object:

_Error in Graph.actor.youtube(yt_actor_network) : object 'g' not found_

Sample (continuing example from above):
```
yt_actor_network <- yt_data |> Create("actor") |> Graph()
```

The error does not occur when using `vosonSML 0.34.3` on `R 4.3.2`.

Not tested for Reddit due to Issue #58 .

**Comment by rjackland** [comment]
@bryn-g already knows this, but I got the same as what @sbinn found: I also got the "object 'g' not found" error for `vosonSML 0.34.3` on `R 4.4.1` on `Ubuntu 22.04`, but it works fine with `R 4.3.3`

**Comment by bryn-g** [comment]
Error is fixed in 5fa70dc.
